src/net.h

```diff
@@ -759,8 +759,8 @@ class CNode
     // socket
     std::atomic<ServiceFlags> nServices{NODE_NONE};
     SOCKET hSocket GUARDED_BY(cs_hSocket);
-    size_t nSendSize{0}; // total size of all vSendMsg entries
-    size_t nSendOffset{0}; // offset inside the first vSendMsg already sent
+    size_t nSendSize GUARDED_BY(cs_vSend){0}; // total size of all vSendMsg entries
```

Comment:
92897d43443db3a7cd68d8655d1bbf4b9b69c427

nit, clang-format suggests:
```suggestion
    size_t nSendSize GUARDED_BY(cs_vSend){0};   // total size of all vSendMsg entries
```